src/components/Search/AdvancedTips/AdvancedTips.js

```diff
@@ -1,175 +1,288 @@
 import './AdvancedTips.less';
 import React from 'react';
+import { TooltipWrapper } from '../../Common/TooltipWrapper/TooltipWrapper';
+import { ComplexExample } from './ComplexExample';
+import iconMap from '../../iconMap';
 
 export const AdvancedTips = () => {
   return (
-    <nav className="advanced_tips">
+    <div className="advanced_tips">
       <div className="o-expandable">
-        <div
-          className="o-expandable_content
-              o-expandable_content__onload-open"
-        >
-          <h3>Advanced search tips</h3>
-          <div className="tips">
-            <div className="tip">
-              <h4>And/Or</h4>
+        <div className="o-expandable_content">
```

Comment:
The structure here looks like it's:
```
o-expandable
  o-expandable_content
  o-expandable_content
  o-expandable_content
```

Expandables were built with the idea it would be one content area per expandable, so it is expecting this structure:

```
o-expandable
  o-expandable_content
o-expandable
  o-expandable_content
o-expandable
  o-expandable_content
```

Additionally, it would expect to have a `o-expandable_target` for toggling the expandable (see https://cfpb.github.io/design-system/components/expandables). Are these used for some expandable styles, but are not actually expandables?

Reply:
Yeah, basically are used for the styling that comes with design system.

Reply:
@anselmbradford 

Could you give this another look?  I'm necromancing this PR from a couple years ago.